src/client/components/equalizer/equalizer.js

```diff
@@ -100,13 +102,45 @@ class Equalizer extends Component {
      } = this.props;
      if (!playPauseState) {
        sound.play();
-     } else {
+     }
+     await playPauseSoundFromFileAsProp();
+     await this.renderEqualizer();
+   }
+
+   pauseSoundFromFile = async () => {
+     const {
+       audioData: {
+         sound,
+         playPauseState,
+       }, playPauseSoundFromFileAsProp,
+     } = this.props;
+     if (playPauseState) {
```

Comment:
Please, add an extra space after this.props

Reply:
done